src/xia2/Handlers/Versions.py

```diff
@@ -63,7 +63,12 @@ def get_aimless_version():
 
 def get_pointless_version():
     result = procrunner.run(
-        ["pointless", "--no-input"], print_stdout=False, print_stderr=False
+        [
+            "pointless",
+        ],
+        print_stdout=False,
+        print_stderr=False,
+        timeout=1,
```

Comment:
This is succeeding by simply _killing_ the pointless process as it waits for `stdin`

Reply:
what's weird is that `pointless` run from the command line does _not_ appear to wait, so I don't know why this happens in i2